docs/troubleshooting.rst

```diff
@@ -4,55 +4,59 @@ Troubleshooting
 NoSuchKernel Errors (using Conda)
 ---------------------------------
 
-NoSuchKernel Errors can appear when running papermill on jupyter notebooks whose
-kernel as been specified via conda (nb_conda). nb_conda is used to easily set
+``NoSuchKernel`` Errors can appear when running papermill on jupyter notebooks whose
+kernel has been specified via conda (nb_conda). nb_conda is used to easily set
 conda environment per notebook from within jupyterlab.
 
-To illustrate, let us create a new environment with all the dependencies
-necessary for an analysis, for example here, a simple R notebook:
+To illustrate, the following example demonstrates the creation of a new
+environment with all the dependencies necessary for an analysis.
 
-``conda create -n analysis_1 r-essentials``
+::
+
+  conda create -n analysis_1 python=2 ipykernel
 
-Once nb_conda is used within the jupyter server to set the kernel for this
+Once nb_conda is used within the jupyter server to set the kernel for a
 notebook to *analysis_1*, the notebook gets metadata similar to the following:
 
-::
-   
-   "metadata": {
+.. code-block:: json
+
+  {
    "kernelspec": {
-       "display_name": "R [conda env:analysis_1]",
-       "language": "R",
-       "name": "conda-env-analysis_1-r"
-   } ...
+    "display_name": "Python [conda env:analysis_1]",
+    "language": "python",
+    "name": "conda-env-analysis_1-py"
+   }
+  }
 
-Using papermill from the *analysis_1* environment or another environment will not work and return the following error:
+Papermill cannot (currently) use this metadata to determine that it should use
```

Comment:
I recommend removing the "(currently)" here since it doesn't. If this should be supported, I recommend making the issue for it then creating a link to it as an anecdote. Otherwise it potentially sounds like we have in progress work to support this.

Reply:
right, now that I'm a member, any problem if I push a commit to master removing that one word without bothering with a branch & PR? Seems overkill.

Reply:
Go for a direct push for that if it lets you for this.

Reply:
I think you'll need to add me to https://github.com/orgs/nteract/teams/papermill

Reply:
Done, glad to have you interested in contributing!

Reply:
Great, all done. Likely to contribute more seeing as I've only just started using papermill.

I feel the "papermill inserts a new cell below the tagged parameter cell so don't put inter-dependent parameters in your tagged cell" isn't really made clear enough. Actually, is it even written anywhere?

e.g.:

----
```python
#parameters
a = 1
b = a * 2
```
```python
print(a, b)
```
----
```bash
$ papermill ... -p a 9
9 2  # not 9 18!
```

Perhaps worth of another issue?

Reply:
It's probably not documented as well as it could be made a little better in that regard. But it's implied by the documentation in the [readthedocs](https://papermill.readthedocs.io/en/latest/usage-parameterize.html?highlight=parameter%20cell#how-do-parameters-work) by indicating there'll be a new cell. A note about inter-dependent calculations would be an improvement there imo.